shared/constants/metametrics.ts

```diff
@@ -512,6 +512,7 @@ export enum MetaMetricsEventName {
   AppLocked = 'App Locked',
   AppWindowExpanded = 'App Window Expanded',
   BridgeLinkClicked = 'Bridge Link Clicked',
+  BtcExperimentalToggled = 'BTC Experimental Toggled',
```

Comment:
I wonder if we should also align this name to `BitcoinSupportToggled`?

Reply:
[f39d57f](https://github.com/MetaMask/metamask-extension/pull/25672/commits/f39d57fe7442c7bf181ad14dcb2acc811cbd6a73)